libsql-sqlite3/tool/mksqlite3internalh.tcl

```diff
@@ -73,6 +73,7 @@ foreach hdr {
    vdbe.h
    vdbeInt.h
    vectorInt.h
+   vectorIndexInt.h
```

Comment:
Should we keep the lexicographical order?

Reply:
Don't think actually. Let's keep them sorted...